aptos-move/framework/src/natives/util.rs

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Aptos
```

Comment:
Hmm?

Reply:
@geekflyer , I think the license check was actually a legitimate lint ;)

Reply:
Oops